src/main/java/duke/Task.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a task which an user input in the <code>Duke</code> chatbox.
+ * A <code>Task</code> consists of 2 components:
+ * <code>instr</code> (instructions), which describes the task required to be carried out and
+ * <code>done</code>, which describes the status of the task (whether it is completed).
+ */
+public class Task {
+    String instr;
+    String type;
+    boolean done;
+
+    public Task(String instr) {
+        this.instr = instr;
+        this.type = "TASK";
+        this.done = false;
+    }
+
+    /**
+     * Changes the <code>Task</code>'s status to <code>done</code> when the task is completed.
+     */
+    public void done() {
```

Comment:
Hmm, instead of having two methods to toggle the done, perhaps could just merge them into one.